### ISSUE TITLE
Creating tunnel... Create tunneling failed

### ISSUE THREAD

**Issue by joacub** [TablePlus/TablePlus#2700]
### Please answer the questions below, it helps us to keep track of the issue. Please do not ignore it or your issue will be closed.

1. Which driver are you using and version of it:
MariaDB
2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
424
3. The steps to reproduce this issue:
Backup -> select connection -> database -> star backup

**Comment by huyphams** [comment]
Hi @joacub it is not possible to reproduce the issue with this info. Are you able to connect to that connection?

**Comment by joacub** [comment]
Yes sir I can connect just the mariadb connections does not work for backups connecting via ssh, I have a MacBook Pro m1.

**Comment by laurentvd** [comment]
@joacub You've probably solved this by now, but I ran into the same issue and fixed it. In my case the IP failed had a trailing space. When hitting the 'test' button, al seemed fine. But when creating a backup, it failed. Removing the space at the end and saving it fixed it for me.

**Comment by pkayokay** [comment]
I ran into this issue as well 🤦 and deleted the extra space I also had in my IP field.

**Comment by simkimsia** [comment]
Same issue here. Had extra space. Please help to trim the IP field
